cmd/insgocc/insgocc.go

```diff
@@ -64,21 +63,6 @@ func (r *outputFlag) Type() string {
 	return "file"
 }
 
-func findPath() (string, error) {
-	gopath := os.Getenv("GOPATH")
-	if gopath == "" {
```

Comment:
I think, this is the pretty legit case and should not be an error.
Probably it's a good idea to use build.Default from `go/build` package (https://golang.org/pkg/go/build/#Context)